### PR TITLE
Added sorting, used post and delete method, and added a navigation bar

### PR DESCRIPTION
**Pull request by Mariela-t**
I deleted the component TodoContainer since it was optional

### CODE REVIEW

**Comment by kwilson05** [comment]
I spoke to Roy. It was def optional and came up out of nowhere so I think its fair to skip it. 👍🏾

File: src/App.js [hunk 1]
```diff
@@ -53,6 +96,10 @@ function App() {
             <div className={styles.container}>
               <h1 className={styles.h1}>Todo list</h1>
               <AddTodoForm onAddTodo={addTodo} />
+              <button type="button" onClick={() => setOrder(!order)}>
+                {" "}
+                button
```
Comment (kwilson05) [inline src/App.js:101]: Maybe call this button 'Sort'. And try to add a way to bring it back its original state.
Also, think about what happens to the sorted list when add a new `todoItem`. The list should probably remain in the sorted order A-Z or Z-A if you add a new item so you'll have to re-sort after each item is added.

File: src/components/TodoListItem.js [hunk 2]
```diff
@@ -6,12 +6,11 @@ function TodoListItem({ item, onRemoveTodo }) {
...
     <>
       <li className={styles.ListItem}>
-        {" "}
-        {item.fields.Title}{" "}
+        {item.fields?.Title ? item.fields.Title : item.title}
         <button
           className={styles.button}
           type="button"
-          onClick={() => onRemoveTodo(item)}
+          onClick={() => onRemoveTodo(item.id)}
```
Comment (kwilson05) [inline src/components/TodoListItem.js:13]: Try to think of a way to show a loading or removing status while removing an item so the user knows the app is actually doing something you remove. You probably do something similar for adding an item :)

**Review by kwilson05 — APPROVED** [review]
Good job here! Left a few notes

File: src/App.js (hunk 1 above)
Comment (Mariela-t) [reply src/App.js:101]: Yes I noticed after I submitted the assignment that I had accidentally left the button like that 🤦🏽‍♀️

**Pull request merged**